doc/ref_impl/cntt-ri/chapters/chapter03.md

```diff
@@ -4,16 +4,56 @@
+  - Instance capabilities do not explicitly appear on the naming convention, because some are common to all instance types so they are covered in the `<Flavour>` (RM1: 4.2.1 Compute Flavours), additionally there are a few capabilities which are bind to certain types, e.g., CPU pinning or NUMA support are only available for N/C instances, while CPU overbooking can only happens B instance but not for N/C. (See RM1: Table 4-24 Mapping of NFVI Capabilities to Instance Types for full mapping details)
+  - `<S ext>` stands for persistent storage extensions, contains the size and the performance settings (RM1: Table 4-20: Storage Extension Options), note the storage extension is common to all instance types as <Flavours>
+  - `<A ext>` stands for accelaration extensions, features like Transcoding and Programmable are associated with C instances (RM1: 4.2.4.3.1 Compute Accleration Extensions), while IPSec and Crypto features only make scene with N instance (RM1: 4.2.4.2.1 Network Acceleration Extensions),
 
+Every VNF instance must declare its profiles explicitly, which can be used by VIM to allocate resources duration instantiation, also it would be useful to evaluate portability of workload.
+
+> Open Point 1: Does ONAP have some relevant spec or VNF declaration schema so that CNTT can re-use/revise to cover what we need ? Or define a new one ?
+> Open Point 2: What princples should be followed if some the pre-define VNF profile items does not match what actual requires ? How to adjust, "ceiling", "floor", "customerize" ?
+
+> Open Point 3: Is it relevant/necessary to include NFVI SW/HW profile description file as following sections ?
 
 <a name="3.3"></a>
-## 3.3 Heading
+## 3.3 NFVI SW profile
+
+[RM1: 5.2 NFVI SW profile features and requirements](https://cntt-n.github.io/CNTT/doc/ref_model/chapters/chapter05.html#5.2) defines the NFVI software layer. The profile depicts the feature status of the 
+  - virtual Compute (**nfvi.com.cfg.xxx** in RM1 Table 5-7: Virtual Compute features and configuration for the 3 types of SW profiles and **nfvi.com.acc.cfg.xxx** in Table 5-8: Virtual Compute Acceleration features),
+  - storage (**nfvi.stg.cfg.xxx** in RM1: Table 5-9: Virtual Storage features and configuration for the 3 types of SW profiles and **nfvi.stg.acc.cfg.xxx** in Table 5-10: Virtual Storage Acceleration features)
+  - networking configuration(see **nfvi.net.cfg.xxx** in Table 5-11 Virtual Networking features and configuration for the 3 types of SW profiles and **nfvi.net.acc.cfg.xxx** in Table 5-12 Virtual Networking Acceleration features)
+
+This profile is the global settings for the whole NFVI, which means there should be only one entry per NFVI resource pool, i.e., Basic/Netowrk/Compute
```

Comment:
Netowrk -> Network

Reply:
I did not find the html anchors for the tables in RM.

Do you have any suggestions on how to add hyperlinks for these tables.

Reply:
Yeah, right. We don't define any anchor for tables as of now. We can link it to the section that contain that table OR skip it altogether. I will prefer skipping it.

Reply:
Hi @deebhatia 

By "skipping" you means **remove the text** or "some tricks" to represent the link ? Any existing doc example, I'd like to have a look first.

The goal is to make myself and readers to be able to navigate back to the original requirement documentation easily.

Reply:
By skipping, I mean keep it as you have added currently i.e. the table name without any hyperlinks.